tests/backends/data/test_base.py

```diff
@@ -0,0 +1,71 @@
+"""Tests for the base data backend"""
+
+import pytest
+
+from ralph.backends.data.base import BaseDataBackend, BaseQuery, enforce_query_checks
+from ralph.exceptions import BackendParameterException
+
+
+@pytest.mark.parametrize(
+    "value,expected",
+    [
+        (None, BaseQuery()),
+        ("foo", BaseQuery(query_string="foo")),
+        (BaseQuery(query_string="foo"), BaseQuery(query_string="foo")),
+    ],
+)
+def test_backends_data_base_enforce_query_checks_with_valid_input(value, expected):
+    """Tests the enforce_query_checks function given valid input."""
```

Comment:
```suggestion
    """Test the enforce_query_checks function given valid input."""
```

And so on for tests :)

Reply:
Well spotted) Thanks) Included this change in the #365 PR